core/src/main/scala/cats/mtl/syntax/ask.scala

```diff
@@ -19,7 +19,10 @@ package mtl
 package syntax
 
 trait AskSyntax {
-  implicit def toReaderOps[E, A](fun: E => A): ReaderOps[E, A] = new ReaderOps(fun)
+  implicit def catsMtlSyntaxToReaderOps[E, A](fun: E => A): ReaderOps[E, A] = new ReaderOps(fun)
+
+  @deprecated("use catsMtlSyntaxToReaderOps", "1.6.1") def toReaderOps[E, A](
```

Comment:
nitpick: separating out the deprecation into its own line will make the diff smaller and the formatting less wonky.